rackcity/models/asset.py

```diff
@@ -131,6 +133,13 @@ class AbstractAsset(AssetID):
     )
     comment = models.TextField(null=True, blank=True)
 
+    cpu = models.CharField(max_length=150, blank=True)
+    display_color = models.CharField(
+        max_length=7, validators=[validate_display_color], blank=True
+    )
+    storage = models.CharField(max_length=150, blank=True)
+    memory_gb = RCPositiveIntegerField(null=True, blank=True)
+
```

Comment:
following up on our convo about how its bad practice to have null = True for CharFields, so i did not include that here. but important to track that this is different from how these fields are implemented in it_model

Reply:
Did you test what happens if you create an asset where these are null?  Does it default to empty string or does it throw an error?  I think the former is preferred but this seems like it would do the latter

Reply:
Also could you change this in ITModel while you're at it?

Reply:
If True, Django will store empty values as NULL in the database. Default is False.
Avoid using null on string-based fields such as CharField and TextField because empty string values will always be stored as empty strings, not as NULL. If a string-based field has null=True, that means it has two possible values for "no data": NULL, and the empty string. In most cases, it’s redundant to have two possible values for "no data"; the Django convention is to use the empty string, not NULL.

https://stackoverflow.com/questions/8609192/differentiate-null-true-blank-true-in-django

Reply:
I believe it defaults to empty string and i did not get any warnings when i migrated but i will double check this